core/models.py

```diff
@@ -147,6 +176,25 @@ def configure_logging(name, level=None):
         logger.addHandler(ch_debug)
 
 
+from string import Formatter
+from difflib import get_close_matches
+from enum import IntEnum
+import _string
+import discord
+from discord.ext import commands
```

Comment:
Please keep imports at the top of the file, not sure why it's moved down here?

Reply:
This is to make sure the logger class is set to `ModmailLogger` (with `colorama` initialized) before `discord` is imported (even in `bot.py). Otherwise the color would not take affect on log text from discord logger and it would just be in white.
And sorry, this was poorly done. I'll try to make this cleaner if possible.

Reply:
I guess colors for Discord logger is not the priority, so I will restore the import order and maybe figure the colors on separate PR.